src/github/entities/user.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Base } from 'src/universal/base.entity';
+import { PullRequest } from 'src/webhook/entities/pull-request.entity';
+import { Repo } from 'src/webhook/entities/repository.entity';
+import { Column, Entity, OneToMany } from 'typeorm';
+import { UserInstallation } from './user-installation.entity';
+@Entity()
+export class User extends Base {
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  name: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  phone: string;
+
+  @Column({ type: 'varchar', length: 255, nullable: true })
+  email: string;
+
+  @Column({ type: 'varchar', nullable: true })
+  password: string;
```

Comment:
PullProofAI

Sensitive fields like `password`, `githubAccessToken`, and `accessToken` should be stored securely. Consider implementing hashing algorithms for passwords and encryption for access tokens.
```typescript
8:   @Column({ type: 'varchar', length: 255, nullable: true })
9:   name: string;
10: 
11:   @Column({ type: 'varchar', length: 255, nullable: true })
12:   phone: string;
13: 
14:   @Column({ type: 'varchar', length: 255, nullable: true })
15:   email: string;
16: 
17:   @Column({ type: 'varchar', nullable: true })
18:   password: string; // Consider using bcrypt for password hashing
```


<!-- This is an auto-generated comment by PullProofAI -->